styles/App.scss

```diff
@@ -1,3 +1,30 @@
 body {
     font-family: 'Montserrat', sans-serif;
 }
+
+
+.movie-tiles {
+    display: flex;
+    flex-flow: column row;
+    gap: 10px;
+    overflow: hidden;
+    justify-content: space-between;
+    .movie-tile {
+
+        text-align: center;
+
+        img {
+            object-fit: contain;
+            height: 32vh;
+            flex-grow: 4;
+        }
+
+        .title {
+            background-color: #78736F;
+            padding: 10px;
+            // text-overflow: ellipsis;
```

Comment:
if we don't need piece of code, we should remove it

Reply:
code has been changed